frontend/packages/integration-tests-cypress/views/snapshot.ts

```diff
@@ -0,0 +1,11 @@
+export namespace SnapshotDetails {
+  export const name = 'dd[data-test-selector="details-item-value__Name"]';
+  export const namespace = 'dd[data-test-selector="details-item-value__Namespace"] a';
+  export const pvc = 'dd[data-test-selector="details-item-value__PVC"] a';
+  export const vsc = 'dd[data-test-selector="details-item-value__VSC"] a';
+  export const sc = 'dd[data-test-selector="details-item-value__SC"] a';
+}
+
+export const pvcDropdown = '#pvcDropdown';
+export const snapshotClassDropdown = '#snapshotDropdown';
+export const dropdownFirstItem = '.dropdown-menu__autocomplete-filter li a';
```

Comment:
Should not be using # ids or class selectors (unless a third party lib where we can't add our own `data-test=...`.
Pls see: https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements